test/Listing.spec.js

```diff
@@ -22,7 +22,7 @@ let listingInfo = {
 };
 
 describe('<Listing />', () => {
-  it('calls componentDidMount', () => {
+  xit('calls componentDidMount', () => {
```

Comment:
You did not add any tests and instead commented one out. If you need to refactor the test thats cool but do not remove a test unless you have a very good reason for it. Also please add tests for this feature.